CHANGELOG.md

```diff
@@ -23,7 +23,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fix bug where running `conjurctl server` or `conjurctl account create` with
   passwords that contain `,`s sent via stdin raised an error.
   [cyberark/conjur#2159](https://github.com/cyberark/conjur/issues/2159)
-- Specify keepalive timeout for puma to be longer than proxy and load balancers
+- Specify keepalive timeout for puma to be longer than proxy and load balancers [cyberark/conjur#2191](https://github.com/cyberark/conjur/pull/2191)
```

Comment:
can this say more about what the user impact of this change is? here is a suggestion from the PR description:
```suggestion
- Update the default keepalive timeout for puma to be longer than most common proxy and load balancers.
  Previously, the load balancer in front of Conjur would commonly have a longer timeout than the
  server itself, which can lead to Conjur closing connections even as there are pending requests and
  the proxy returning 502 errors to the client.
  [PR cyberark/conjur#2191](https://github.com/cyberark/conjur/pull/2191)
```